docs/ja/breaking_changes_instructions.md

```diff
@@ -0,0 +1,48 @@
+# breaking changes/互換性を破る変更: プルリクエストにフラグが付けられた
+
+<!---
+  grep --no-filename "^[ ]*git diff" docs/ja/*.md | sh
+  original document: 0.8.75:docs/breaking_changes_instructions.md
+  git diff 0.8.75 HEAD -- docs/breaking_changes_instructions.md | cat
+-->
+
+QMK のメンバーがあなたのプルリクエストに返信し、あなたの提出したものは Breaking change (互換性を破る変更) であると述べている場合があります。メンバーの判断では、あなたが提案した変更は QMK やその利用者にとってより大きな意味を持つと考えられます。
+
+プルリクエストにフラグが立てられる原因となるものには、以下のようなものがあります:
+
+- **ユーザーのキーマップに対する編集**
+  ユーザーが自分のキーマップを QMK に提出した後、しばらくしてさらに更新してプルリクエストを開いたところ、それが `qmk/qmk_firmware` リポジトリで編集されていたためにマージできなかったことに気づくことがあるかもしれません。すべてのユーザーが Git や GitHub を使いこなせるわけではないので、ユーザー自身で問題を修正できないことに気づくかもしれません。
+- **期待される動作の変更**
+  QMK の動作を変更した場合、既存の QMK 機能への変更を組み込んだ新しいファームウェアをフラッシュし、希望する動作を復元する手段がないことに気付いた場合、ユーザーはハードウェアまたは QMK が壊れていると考えるようになる可能性があります。
```

Comment:
@noroadsleft, what do you mean by this sentence?

```
- **Changes to Expected Behavior**
  Changes to QMK behavior may cause users to believe their hardware or QMK is broken if they flash new firmware that incorporates changes to existing QMK features, and find themselves without a means to restore the desired behavior.
```

We each have different interpretations of this statement and I'm not sure which one is correct.
I would appreciate it if you could break up the sentences in a way that is easy to understand.

Reply:
> Changes to QMK behavior may cause users to believe their hardware or QMK is broken if they flash new firmware that incorporates changes to existing QMK features

#6480 was an example of the above. The inactivity timeout for RGB Matrix used to be based on ticks, rather than (milli)seconds. That PR as originally submitted changed the time unit used, such that firmware without that changeset applied had the timeout intended by the user, but firmware generated by the same keymap files that incorporated that PR's changes had a timeout that was 1/1200th of the time.

- keymap files without PR 6480 applied: value of `100` results in a timeout of approximately two minutes
- the same keymap files with PR 6480 applied: value of `100` results in a timeout of 0.1 seconds (100ms)

> and find themselves without a means to restore the desired behavior.

The above PR did not originally include a way for the user to restore their intended behavior without reverting that PR. This was later solved (in the same PR) by changing the name of the define used for the feature. (The code checks for the presence of the old define and the absence of the new define, and converts the former to the latter in those instances.)

----

Without some instruction that tells the user "behavior X of feature Y has changed," a user may flash new firmware to their board, notice the change in behavior, and believe that either QMK or their keyboard is broken in some way.

Does this answer your question sufficiently?